sdk/identity/azure-identity/src/main/java/com/azure/identity/implementation/IdentityClientBase.java

```diff
@@ -619,6 +620,7 @@ AccessToken getTokenFromAzureDeveloperCLIAuthentication(StringBuilder azdCommand
             }
 
             ProcessBuilder builder = new ProcessBuilder(starter, switcher, azdCommand.toString());
+            builder.redirectInput(ProcessBuilder.Redirect.from(IdentityUtil.NULL_FILE));
```

Comment:
1. could you add a comment here mentioning why we're doing the redirection
2. could you include a test case in the PR for the scenario